react/wbooks/package.json

```diff
@@ -6,6 +6,7 @@
     "apisauce": "^1.1.1",
     "cerealizr": "^1.0.0-alpha",
     "i18next": "^13.1.5",
+    "local-storage": "^2.0.0",
```

Comment:
for this case this is useless, same here as lodash. LocalStorage API is too easy. go check it!